src/index.ts

```diff
@@ -113,7 +119,7 @@ async function main(): Promise<string[] | undefined> {
     });
   }
 
-  if (build == null) {
+  if (build === null) {
```

Comment:
Don't do this

`== null` catches `undefined` and `null`
```suggestion
  if (build == null) {
```

Reply:
Fine to go this way but...

IMO it'd be better to be explicit that you're checking both instead of relying on the implicit knowledge that `==` checks both. Also will build ever be undefined? We type it as `Build | null` so in theory it won't be.

Either way, could we assume build isn't a boolean/string/number and do

```ts
if(!build) { ... }
```